modules/actions/functions.js

```diff
@@ -0,0 +1,151 @@
+import { createAction } from './utils';
+import AppConstants from '../constants';
+import {
+	getFunctions as fetchFunctions,
+	updateFunctions as putFunctions,
+	createFunction as deployFunction,
+	invokeFunction as invokeFunctions,
+	deleteFunction as deleteFunctions,
+	reorderFunction as reorderFunctions,
+	getSingleFunction as getFuncInfo,
+} from '../../utils/app';
+
+export function getFunctions(name = 'default') {
+	return dispatch => {
+		dispatch(createAction(AppConstants.APP.FUNCTIONS.GET));
+		return fetchFunctions()
+			.then(res => {
+				dispatch(
+					createAction(AppConstants.APP.FUNCTIONS.GET_SUCCESS, res, null, {
+						name,
```

Comment:
No need to use `name` here since it's a cluster-specific feature, not an `index` level.